### DISCUSSION TITLE
Is there any possibility for courseplay to unload forage harvesters, the same way CP can unload combines?

### DISCUSSION THREAD

**Discussion by daantimmer** [Courseplay/Courseplay_FS22#2446] [Q&A] [answered]
I sometimes find the AD combine unload to be, less optimal. And sometimes I then switch to using CP's combine unload function instead. But this doesn't seem to work, at all, for forage harvesters.

Is this intended? All that happens is the unloader drives to the "target position" location, and then just waits indefinitely.

Kind regards,

**Comment by Tensuko** [comment, marked as answer]
No and not planed.

**Reply by daantimmer** [reply]
Right. Thanks for the clear answer. I wasn't sure if I was doing something wrong or whether it was never added.
